stdlib/LibGit2/src/tree.jl

```diff
@@ -122,14 +128,13 @@ tree_entry = tree[1]
 blob = LibGit2.GitBlob(tree_entry)
 ```
 """
-function GitObject(e::GitTreeEntry) end
```

Comment:
This function was declaring this signature to return nothing, which is obviously wrong. How do I declare it to just document the signature? @fredrikekre or @KristofferC do you know?

Reply:
just `GitObject(e::GitTreeEntry)` I believe.

Reply:
Thanks.